ddsc/core/ddsapi.py

```diff
@@ -115,7 +200,7 @@ def _check_err(self, resp, url_suffix, data):
         :param data: data payload we sent
         :return: requests.Response containing the successful result
         """
-        if resp.status_code != 200 and resp.status_code != 201:
```

Comment:
Is there a reason to limit to just these 3 2xx codes? Why not `int(resp.status_code / 100) != 2'